outputs/report.html

```diff
@@ -0,0 +1,332 @@
+<!DOCTYPE html>
```

Comment:
It took me some digging to figure out what report.html was for. Could it be renamed to something like "sample_report.html" or something to make its purpose clearer, please?

Reply:
Possibly, everything in `outputs` is a sample though!